.vscode/extensions.json

```
@@ -0,0 +1,7 @@
+{
+    "recommendations": [
+        "esbenp.prettier-vscode",
+        "johnsoncodehk.vscode-vue-preview",
+        "ddbaeumer.vscode-eslint"
```

I guess this should rather be`?
```suggestion
        "dbaeumer.vscode-eslint"
```